setup.cfg

```
@@ -31,6 +31,7 @@ install_requires =
     tables>=3.2.1
     pybids~=0.15.4
     jinja2
+    pymare~=0.0.3
```

We'll want to update other dependencies later, but I've been convinced that adding upper bounds (`~=0.0.3` == `>=0.0.3,<0.1`) is a bad idea.

```suggestion
    pymare>=0.0.3
```